src/NewRelic.Telemetry/Logging/TelemetryLogging.cs

```diff
@@ -14,27 +14,27 @@ public class TelemetryLogging
         private const string Prefix = "NewRelic Telemetry:";
         private const string Category = "NewRelic.Telemetry";
 
-        private readonly ILogger _logger;
+        private readonly ILogger? _logger;
```

Comment:
I think we can make this non-nullable if we use the [`NullLogger`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.logging.abstractions.nulllogger) in the event that someone has initialized things without a `LoggerFactory`.